cli/standalone.rs

```diff
@@ -106,28 +139,28 @@ impl ModuleLoader for EmbeddedModuleLoader {
   }
 }
 
-async fn run(source_code: String, args: Vec<String>) -> Result<(), AnyError> {
-  let flags = Flags {
-    argv: args[1..].to_vec(),
-    // TODO(lucacasonato): remove once you can specify this correctly through embedded metadata
-    unstable: true,
-    ..Default::default()
-  };
+async fn run(source_code: String, flags: Flags) -> Result<(), AnyError> {
   let main_module = ModuleSpecifier::resolve_url(SPECIFIER)?;
-  let permissions = Permissions::allow_all();
+  let permissions = Permissions::from_options(&flags.clone().into());
   let module_loader = Rc::new(EmbeddedModuleLoader(source_code));
   let create_web_worker_cb = Arc::new(|_| {
     todo!("Worker are currently not supported in standalone binaries");
   });
 
+  v8_set_flags(
```

Comment:
Please add a note that it should be kept in sync with the identical call in `main.rs`